packages/server/types/create-instance.d.ts

```diff
@@ -10,10 +10,16 @@ export interface EmotionCritical {
   css: string
 }
 
+export interface EmotionCriticalToChunks {
+  html: string
+  styles: Array<{ key: string; ids: Array<string>; css: string }>
+}
+
 export interface EmotionServer {
   extractCritical(html: string): EmotionCritical
   renderStylesToString(html: string): string
   renderStylesToNodeStream(): NodeJS.ReadWriteStream
+  constructStyleTags(html: string): EmotionCriticalToChunks
```

Comment:
This is not correct. The return type of this is a string but the interface definition is missing a declaration for `extractCriticalToChunks` - for which this return type would be correct.

wdyt about renaming `constructStyleTags` to `constructStyleTagsFromChunks`? That way it would be more clear that those 2 APIs can be used together (`extractCriticalToChunks` and `constructStyleTagsFromChunks`)